tests/test_sequence.py

```diff
@@ -0,0 +1,83 @@
+import os
+import unittest
+import logging
+from copy import copy
+from pathlib import Path
+
+# required for tensorflow logging control
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+import tensorflow as tf
+import numpy as np
+import enso
+from sklearn.metrics import accuracy_score
+import requests
+from finetune import LanguageModelGeneralAPI
+
+from bs4 import BeautifulSoup as bs
```

Comment:
Add to deps if we're going to use bs4.  Or alternately use the xml lib in the tests.